data-explorer/power-bi-imported-query.md

```diff
@@ -22,6 +22,7 @@ You need the following to complete this article:
 
 * An organizational email account that is a member of Azure Active directory, so you can connect to the [Azure Data Explorer help cluster](https://dataexplorer.azure.com/clusters/help/databases/samples).
 * [Power BI Desktop](https://powerbi.microsoft.com/get-started/) (select **DOWNLOAD FREE**).
+* [Power BI Desktop - Alternate Download Link](https://www.microsoft.com/en-us/download/details.aspx?id=58494).
```

Comment:
```suggestion
* [Power BI Desktop - Alternate Download Link](https://www.microsoft.com/download/details.aspx?id=58494).
```

Reply:
If the submission is accepted, remove the locale identifier (en-us) from the URL.